### PR TITLE
Added responsive navbar styling.

### PR DESCRIPTION
**Pull request by wavpack**
Navbar is now sticky and responsive.
Navbar Logo has a highlight.
Navbar Toggler and Dropdowns have borders and highlights.

**IMPORTANT**: May have to resolve conflicts in...
- app/views/pages/_find_resto.html.erb

### CODE REVIEW

**Comment by wavpack** [comment]
[style the navbar](https://trello.com/c/gdOYz8VS/104-style-the-navbar)

**Comment by wavpack** [comment]
May have to resolve conflicts in:
- app/views/pages/_find_resto.html.erb

**Comment by wavpack** [comment]
Preview Pictures:

navbar-signout
navbar-signin

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -21,13 +23,13 @@
     <meta name='twitter:image:alt' content='food-fluent-logo' />
     <meta name='twitter:title' content='food-fluent' />
     <meta name='viewport' content='width=device-width,initial-scale=1,shrink-to-fit=no' />
-    <meta property='og:title' content='food-fluent' />
     <meta property='og:description' content='Find a restaurant for everyone on food-fluent' />
-    <meta property='og:type' content='website' />
-    <meta property="og:url" content="<%= request.original_url %>" />
-    <meta property="og:image" content="<%= 'meta_image.png' %>" />
-    <meta property='og:site_name' content='food-fluent' />
```
Comment (sofofos) [inline app/views/layouts/application.html.erb:29]: the order was changed to pass the product page's checks, can you remove this file change from your pull request pls

File: app/views/pages/_find_resto.html.erb [hunk 2]
```diff
@@ -1,39 +1,53 @@
...
-    Find a restaurant!
-  </h5>  <!-- end dash-finder-titlebar-left-text !-->
+<div class='row' id='dash-finder-row'>
+  <div class='row' id='dash-finder-titlebar'>
+    <div class='column' id='dash-finder-titlebar-left'>
+      <h5 id='dash-finder-titlebar-left-text'>
+        Find a restaurant!
+      </h5>  <!-- end dash-finder-titlebar-left-text !-->
+    </div> <!-- end dash-finder-titlebar-left !-->
+  </div> <!-- end dash-finder-titlebar !-->
```
Comment (sofofos) [inline app/views/pages/_find_resto.html.erb:9]: we're changing find_resto behaviour so this file wont be needed, pls check my branch prep-deploy for the updates. as such this file should also be removed from the pull request

**Review by sofofos — CHANGES_REQUESTED** [review]
need to exclude two files from pull request to approve the navbar styling

**Comment by sofofos** [comment]
Closing since duplicate of PR84

**Pull request closed without merging**